roles/openshift_logging_fluentd/defaults/main.yml

```diff
@@ -48,7 +48,8 @@ openshift_logging_fluentd_aggregating_strict: "no"
 openshift_logging_fluentd_aggregating_cert_path: none
 openshift_logging_fluentd_aggregating_key_path: none
 openshift_logging_fluentd_aggregating_passphrase: none
-openshift_logging_use_mux_client: False
+# this tells the fluentd node agent to use mux instead of sending directly to Elasticsearch
+#openshift_logging_mux_client_mode: "full_no_k8s_meta" or "minimal"
```

Comment:
the code is checking if its defined so setting it to `None` makes it defined... commenting it out is valid IMHO